MCPForUnity/UnityMcpServer~/src/tools/manage_gameobject.py

```
@@ -113,6 +114,13 @@ def _to_vec3(parts):
     search_inactive = _coerce_bool(search_inactive)
     includeNonPublicSerialized = _coerce_bool(includeNonPublicSerialized)
 
+    # Coerce 'component_properties' from JSON string to dict for client compatibility
+    if isinstance(component_properties, str):
+        try:
+            component_properties = json.loads(component_properties)
+            ctx.info("manage_gameobject: coerced component_properties from JSON string to dict")
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)